### ISSUE TITLE
Elevated plasma tetradecenolycarnitine, C14:2

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5973]
**Preferred term label:** Elevated plasma tetradecenolycarnitine, C14:2

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Tetradecenolycarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
See https://github.com/obophenotype/human-phenotype-ontology/issues/5971
